android/app/src/main/cpp/app/organicmaps/Framework.cpp

```diff
@@ -1184,14 +1184,14 @@ Java_app_organicmaps_Framework_nativeDisableFollowing(JNIEnv * env, jclass)
 }
 
 JNIEXPORT jobjectArray JNICALL
-Java_app_organicmaps_Framework_nativeGenerateNotifications(JNIEnv * env, jclass)
+Java_app_organicmaps_Framework_nativeGenerateNotifications(JNIEnv * env, jclass, bool shouldAnnounceStreetNames)
```

Comment:
nit: _announceStreets_ var name is better here IMO. No need in these long sentences :)
Also setAnnounceStreets(), shouldAnnounceStreets() functions.